### DISCUSSION TITLE
S0-Zähler Aktualisierungsintervall

### DISCUSSION THREAD

**Discussion by AMI70git** [camueller/SmartApplianceEnabler#516] [Allgemein]
Hallo zusammen,
die Werte der S0-Zähler werden über MQTT nur alle 20 Sekunden aktualisiert.
Kann man dieses Aktualisierungsintervall irgendwo konfigurieren (zum Beispiel auf 10 Sekunden)?

**Comment by camueller** [comment]
Bei geringer Leistungsaufnahme ist die Dauer zwischen zwei Impulsen länger als 60s. Was soll da das häufigere Versenden der MQTT-Nachrichten bringen?

**Comment by AMI70git** [comment]
Hallo Herr Müller,
die Werte werden bei uns in einer Oberfläche zusammen mit Werten aus anderen Quellen angezeigt. Die S0-Zähler sollen den Gesamt-Verbrauch im Detail erklären. Alle anderen Werte werden quasi in Echtzeit angeliefert, und Nutzer haben bemängelt das wenn beispielsweise  der Verbrauch hochgeht die Werte der Einzelzähler so stark hinterherhinken. Es passiert ja erstmal 20 Sekunden nichts. In der Zwischenzeit ist vielleicht ein anderer Verbraucher angesprungen oder ausgegangen. Aus Sicht der Nutzer, von denen ich ja möchte das sie ihren Verbrauch(er) einordnen und verstehen, springen die Werte unzusammenhängend hin und her weil die S0-Zähler so hinterherhinken.
Deswegen wäre es sehr hilfreich wenn man das Intervall irgendwo konfigurieren könnte.

**Comment by camueller** [comment]
Es ist nicht vorgesehen, dieses Intervall konfigurierbar zu machen, zumal der Sunnuy Home Manger ja auch nur alle 60s die Werte abfragt. S0-Zähler sind für eine hohe zeitliche Auflösung auch komplett ungeeignet.

**Comment by AMI70git** [comment]
Mit dem SAE lassen sich ja viele Quellen zusammenbringen außerhalb des SAE, hilfreich wäre es. Aber danke für die Rückmeldung.

**Comment by camueller** [comment]
> die Werte werden bei uns in einer Oberfläche zusammen mit Werten aus anderen Quellen angezeigt

Was für eine Oberfläche ist das? Link?

**Comment by AMI70git** [comment]
Das ist eine selbst entwickelte Software welche über MQTT und Modbus mit unserer Haustechnik kommuniziert.
